### PR TITLE
added DAO methods

### PR DESCRIPTION
**Pull request by boroda4436**

### CODE REVIEW

File: src/test/java/core/basesyntax/dao/animal/AnimalDaoImplTest.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.dao.animal;
+
+public class AnimalDaoImplTest {
+
+}
```
Comment (KseniiaMakarova) [inline src/test/java/core/basesyntax/dao/animal/AnimalDaoImplTest.java:5]: you will add tests later, right?

File: src/main/java/core/basesyntax/model/ma/Person.java [hunk 2]
```diff
@@ -1,6 +1,7 @@
 package core.basesyntax.model.ma;
 
 public class Person {
+    private Long id;
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/model/ma/Person.java:4]: should we add id field to other base classes as well?

**Review by KseniiaMakarova — APPROVED** [review]

**Review by dmytro-hryhoruk — APPROVED** [review]

File: src/test/java/core/basesyntax/dao/animal/AnimalDaoImplTest.java (hunk 1 above)
Comment (boroda4436) [reply src/test/java/core/basesyntax/dao/animal/AnimalDaoImplTest.java:5]: yes

File: src/main/java/core/basesyntax/model/ma/Person.java (hunk 2 above)
Comment (boroda4436) [reply src/main/java/core/basesyntax/model/ma/Person.java:4]: in the next PR I will add it. This is required for the test cases

**Pull request merged**